src/Microsoft.AspNetCore.Razor/Tokenizer/Internal/CSharpKeywordDetector.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Tokenizer.Symbols;
 
-namespace Microsoft.AspNetCore.Razor.Tokenizer
+namespace Microsoft.AspNetCore.Razor.Tokenizer.Internal
 {
     internal static class CSharpKeywordDetector
```

Comment:
Something's up with my box. The namespace should have been nuked. Running through this all locally again.